manifest.json

```diff
@@ -1,20 +1,22 @@
 {
-    "packaging_format": 1,
     "id": "my_webapp",
     "name": "Custom Webapp",
+    "packaging_format": 1,
     "description": {
         "en": "Custom Web app with SFTP access",
         "fr": "Application Web personnalisée avec accès SFTP"
     },
-    "license": "free",
+    "version": "1.0",
+    "url": "https://github.com/YunoHost-Apps/my_webapp_ynh",
+    "license": "GPLv3",
```

Comment:
If I look at the [official syntax](https://spdx.org/licenses/), it should be "GPL-3.0" (yes, I haven't tought about it in previous reviews... :confused:)